src/core/lib/event_engine/iomgr_engine/ev_epoll1_linux.cc

```diff
@@ -77,17 +91,24 @@ class Epoll1EventHandle : public EventHandle {
     write_closure_->InitEvent();
     error_closure_->InitEvent();
     pending_actions_ = 0;
+    exec_actions_closure_ = std::make_unique<Epoll1ExecActionsClosure>(this);
```

Comment:
I don't think the custom Closure type is necessary, nor the forward declaration / circular dependency. If you use something like a basic lambda-wrapped closure, which we've written a few times now, I think this gets simpler. For example:

```
class Epoll1EventHandle {
...
 private:
  AnyInvocableClosure exec_actions_closure_;
};
```

Then in your constructor
`Epoll1EventHandle(...) : exec_actions_closure_([this] { ExecutePendingActions(); Unref(); } )`

Or

`Epoll1EventHandle(...) : exec_actions_closure_([this] { DoCallback(); } )`

I extracted some common, tested closure types I've hacked together a few times (including in my poller) and added them here https://github.com/grpc/grpc/pull/30438. Please review, and we can both use them.

Reply:
Done. I used AnyInvocableClosure now.